packages/compass/package.json

```diff
@@ -164,6 +164,7 @@
   "dependencies": {
     "@mongosh/node-runtime-worker-thread": "^1.5.4",
     "clipboard": "^2.0.6",
+    "compass-preferences-model": "^2.0.0",
```

Comment:
This section only lists externalized dependencies, which `compass-preferences-model` is not – is there a special reason for moving this to regular `dependencies`?